### PR TITLE
docs: Add warning for ldap in zitadel cloud

### PR DESCRIPTION
**Pull request by hifabienne**
```[tasklist]

### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

File: docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+:::caution
+LDAP does send your user passwords in clear text. Due to this we do not recommend using it with the ZITADEL cloud.
+If you want to use it anyway, please reach out to us.
```
Comment (fforootd) [inline docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx:3]:
```suggestion
LDAP does send your passwords in clear text. Due to this we do not recommend using it with the ZITADEL cloud.
If you want to use it anyway, please reach out to us.
```

Comment (fforootd) [reply docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx:3]: Maybe we should quickly elaborate ldap vs ldaps

File: docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+:::caution
+LDAP does send your user passwords in clear text. Due to this we do not recommend using it with the ZITADEL cloud.
```
Comment (eliobischof) [inline docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx:2]: This doesn't apply to local LDAP, right?
Because it's imported there.

Comment (hifabienne) [reply docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx:2]: you should not use a local ldap with zitadel cloud

Comment (hifabienne) [reply docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx:2]: to show the user that this should not be done

File: docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx (hunk 1 above)
Comment (eliobischof) [reply docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx:3]: Do we support ldaps or just STARTTLS?

Comment (eliobischof) [reply docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx:3]: What about this? https://github.com/zitadel/zitadel/pull/5677/commits/d0d7cc6cc2a6feff7ba128d3148b7b3ecbd6116a

**Comment by hifabienne** [comment]
@eliobischof I think we can close this one, as we have it included in your pr, right?

**Comment by eliobischof** [comment]
> @eliobischof I think we can close this one, as we have it included in your pr, right?

If you like my suggestion, yes 🙂

**Pull request closed without merging**